### PR TITLE
Drop MPPTools in samples + migrate to KTS

### PR DESCRIPTION
**Pull request by ddolovov**

### CODE REVIEW

File: samples/csvparser/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,28 @@
+import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTargetPreset
+
+plugins {
+    kotlin("multiplatform")
+}
+
+// Determine host preset.
+val hostOs = System.getProperty("os.name")
+
+val hostPreset: KotlinNativeTargetPreset = when {
```
Comment (olonho) [inline samples/csvparser/build.gradle.kts]: One could do `when (val hostOs = System.getProperty("os.name"))`  I guess.

File: samples/echoServer/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+val hostPreset: KotlinNativeTargetPreset = when {
+    hostOs == "Mac OS X" -> "macosX64"
+    hostOs == "Linux" -> "linuxX64"
+    hostOs.startsWith("Windows") -> "mingwX64"
+    else -> throw GradleException("Host OS '$hostOs' is not supported in Kotlin/Native $project.")
+}.let {
+    kotlin.presets[it] as KotlinNativeTargetPreset
+}
+
+// Add two additional presets for Raspberry Pi.
```
Comment (olonho) [inline samples/echoServer/build.gradle.kts]: for Raspberry Pi and Linux/ARM.

File: samples/gtk/build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+import org.jetbrains.kotlin.konan.target.KonanTarget.*
+
+plugins {
+    kotlin("multiplatform")
+}
+
+// Determine host preset.
+val hostOs = System.getProperty("os.name")
+
+val hostPreset: KotlinNativeTargetPreset = when {
```
Comment (olonho) [inline samples/gtk/build.gradle.kts]: BTW, why don't we have this function in Gradle plugin?

Comment (ddolovov) [reply samples/gtk/build.gradle.kts]: @ilmat192 would you add this to the backlog?

Comment (ilmat192) [reply samples/gtk/build.gradle.kts]: Ok

**Pull request merged**